### ISSUE TITLE
Mcrouter Operator: Can't access logs, metrics, and container shell

### ISSUE THREAD

**Issue by mamurak** [openshift-labs/learn-katacoda#1284]
**In which scenario did this bug occur?**

Mcrouter Operator powered by Ansible Operator

**At what step of the scenario did this happen?**

Steps 1, 2, 3

**Describe the bug**

***No access to logs.***

The scenario instructs the user to display container logs using `oc logs` (steps 1 and 2). However, these commands fail with this error message:
`error: You must be logged in to the server (the server has asked for the client to provide credentials ( pods/log [pod-name]))`

***Can't attach to containers.***

The scenario instructs the user to test Mcrouter connectivity and performance using test containers (step 3). However, the given commands `oc run -it` fail with this error message:
```
Error attaching, falling back to logs: unable to upgrade connection: Unauthorized
pod "telnet" deleted
error: You must be logged in to the server (the server has asked for the client to provide credentials ( pods/log telnet))
```

***No access to metrics.***

The scenario instructs the user to review the metrics dashboard while load testing the Mcrouter pods. However, the metrics dashboard does not display any data points:

![210811_metrics](https://user-images.githubusercontent.com/68942736/128995216-30e4967a-f6ad-4f6e-bdb3-a8b5e7f78396.png)

**Comment by madorn** [comment]
Please see https://github.com/openshift-labs/learn-katacoda/issues/1276

**Comment by madorn** [comment]
Fixed in #1290.

**Comment by jdob** [comment]
Closing per Matt's comment.
